source/_docs/blueprint/selectors.markdown

```diff
@@ -317,6 +323,37 @@ entity:
   device_class: motion
 ```
 
+## Example entity selector for State trigger
+
+```yaml
+blueprint:
+  name: State Entity ID Template
+  description: Demonstrate using a selector to select multiple entities for a State Trigger
+  domain: automation
+  source_url: https://github.com/home-assistant/core/blob/dev/homeassistant/components/automation/blueprints/example.yaml
+  input:
+    entities:
+      name: Trigger Entities
+      selector:
+        target:
```

Comment:
I think this is a misleading example, as it assumes `targets` are only entities, which is incorrect. I don't think we should add this example. Nor should this even be used by someone this way.

Reply:
All Selector.markdown file changes have been removed